extensions/log.py

```diff
@@ -80,6 +82,26 @@ def load_directory(self):
         else:
             return settings['root_directory']
 
+    @commands.command()
+    @commands.has_permissions(administrator=True)
+    async def logs(self, ctx, type='Invalid', amount='100'):
+        if type != 'general' and type != 'error' and type != 'command_error' or (amount != 'all' and not amount.isdigit()):
+            await ctx.send('Usage: `$logs (general | error | command_error) [amount=100 | all]`')
+            return
+        with open(f'{self.root_dir}{type}.log', 'r') as file:
+            if amount == 'all':
+                await asyncio.sleep(0.1) # This is required, as this command executes something is
+                                            # written to the log file, which is temporarily cleared or
```

Comment:
uh

Reply:
I've seen worse.

Reply:
We'll be moving to Discord.js, so temporarily this will suffice.